api/post_test.go

```diff
@@ -615,7 +613,7 @@ func TestSearchPostsFromUser(t *testing.T) {
 		t.Fatalf("wrong number of posts returned %v", len(result.Order))
 	}
 
-	if result := Client.Must(Client.SearchPosts("from: " + user2.Username + " in:" + channel1.Name)).Data.(*model.PostList); len(result.Order) != 1 {
+	if result := Client.Must(Client.SearchPosts("from: " + user2.Username + " in:" + channel1.Name)).Data.(*model.PostList); len(result.Order) != 0 {
```

Comment:
Can you manually add some posts in place of the system messages for this test instead of knocking the expected numbers down? Some of the tests don't check anything right now like this one which was supposed to check if from: and to: can be used together.